examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -0,0 +1,235 @@
+             'Left-Amygdala',
+             'Left-Caudate',
+             'Left-Hippocampus',
+             'Left-Pallidum',
+             'Left-Putamen',
+             'Left-Thalamus-Proper',
+             'Left-Cerebellum-Cortex',
+             'Brain-Stem',
+             'Right-Accumbens-area',
+             'Right-Amygdala',
+             'Right-Caudate',
+             'Right-Hippocampus',
+             'Right-Pallidum',
+             'Right-Putamen',
+             'Right-Thalamus-Proper',
+             'Right-Cerebellum-Cortex']
+'''
+# Here we select only these sub structures we want to include in the source
+# space
+labels_vol = ['Left-Amygdala',
+              'Left-Thalamus-Proper',
+              'Left-Cerebellum-Cortex',
+              'Right-Amygdala',
+              'Right-Thalamus-Proper',
+              'Right-Cerebellum-Cortex']
+
+# Setup a surface-based source space
+src = setup_source_space(sbj_id, subjects_dir=sbj_dir,
+                         spacing='oct6', add_dist=False, overwrite=True)
+
+# We create a mixed src space adding to the surface src space the volume
+# regions specified in the list labels_vol. First, read the aseg file and the
+# source space bounds using the inner skull surface
+
+# Generate the mixed source space
+for l in labels_vol:
+    # setup a volume source space of the label l
+    vol_src = setup_volume_source_space(sbj_id, mri=fname_aseg,
+                                        pos=5.0,
+                                        bem=fname_model,
+                                        volume_label=l,
+                                        subjects_dir=sbj_dir)
+    # combine the source spaces
+    src += vol_src
+
+n = sum(src[i]['nuse'] for i in range(len(src)))
+print('the src space contains %d spaces and %d points' % (len(src), n))
+
+# Write the mixed source space
+write_source_spaces(fname_mixed_src, src)
+
+# Export source positions to nift file
+nii_fname = op.join(bem_dir, '%s-mixed-src.nii' % sbj_id)
+src.export_volume(nii_fname, mri_resolution=True)
+
+# Uncomment the following lines to display source positions in freeview.
+'''
+# display image in freeview
+from mne.utils import run_subprocess
+mri_fname = op.join(sbj_dir, sbj_id, 'mri/brain.mgz')
+run_subprocess(['freeview', '-v', mri_fname, '-v',
+                '%s:colormap=lut:opacity=0.5' % aseg_fname, '-v',
+                '%s:colormap=jet:colorscale=0,2' % nii_fname, '-slice',
+                '157 75 105'])
+'''
+
+# Now we compute the lead field matrix for the mixed source space and the
+# connectivity btw 74 regions in src space (68 in the surface src space and 6
+# in the sub structures) based on dSPM inverse solutions
+
+# Compute the fwd matrix
+fwd = make_forward_solution(fname_raw, fname_trans, src, fname_bem,
+                            fname_fwd,
+                            mindist=5.0,  # ignore sources<=5mm from innerskull
+                            meg=True, eeg=False,
+                            n_jobs=2,
```

Comment:
don't use n_jobs=2 in examples. Set it to 1. Otherwise it causes issues on windows